README.md

```diff
@@ -13,4 +13,4 @@ This is from oniontest2. But commit shows it's from oniontest1... Be careful.
 
 Edit from oniontest1:
 ---
-I'd suggest practicing git using multiple accounts.
\ No newline at end of file
+I'd suggest practicing git using multiple accounts. Use different browser and use local user.name, user.email. Then add SSH Host info to `~/.ssh/config`
```

Comment:
Detailed info for handling multiple accounts added.

Reply:
I think it's good.

Reply:
That looks perfect - I'm 👍 to 🚢 this!